package.json

```diff
@@ -118,6 +119,12 @@
     "debug": "^4.0.0",
     "underscore": "^1.12.1",
     "d3-color": "^3.1.0",
-    "log4js": "^6.4.0"
+    "log4js": "^6.4.0",
+    "react-dnd": "^16.0.1",
```

Comment:
looks like `react-dnd@16.0.1` is the version that `iot-app-kit/*` v7 uses so i don't think we need the resolution here.

Reply:
Good catch!